### PR TITLE
#18 Include path exceptions at end of cp

### PR DESCRIPTION
**Pull request by atdixon**
I necessarily need to depend on some modules that have classes in the default java package. Such modules cause exceptions during path resolution, when javafx-maven-plugin sorts out the modulepath and classpath. Here we include such path exceptions at the tail of the generated classpath.

### CODE REVIEW

**Comment by jperedadnr** [comment]
Hi, @atdixon, thanks for your contribution.
 
Please create an issue for this PR, and provide more details of what is the problem and what should be needed to fix it (that justifies your commit).

**Comment by atdixon** [comment]
Here you go: https://github.com/openjfx/javafx-maven-plugin/issues/18

This is likely worth some discussion. Let me know if I can better articulate my scenario.

**Comment by atdixon** [comment]
A simple demo of the issue can be seen here: https://github.com/atdixon/demo-ywglmcxj/tree/master

**Comment by atdixon** [comment]
Any thoughts on this one?

**Comment by jperedadnr** [comment]
Sorry for not getting back to you sooner.
Please add a short explanation for the new option in the readme (jlink:run options), and we can merge the PR.

**Comment by atdixon** [comment]
Done. Let me know if you there's anything else needed. Thank you.

File: README.md [hunk 1]
```diff
@@ -99,6 +99,8 @@ Optionally, the configuration can be modified with:
 - `outputFile` File to redirect the process output
 - `options`: A list of VM options passed to the executable.
 - `commandlineArgs`: Arguments separated by space for the executed program
+- `includePathExceptionsInClasspath`: Include in classpath even those dependencies with path exceptions encountered
+ during module-path resolution. Values false (default), true
```
Comment (jperedadnr) [inline README.md]: Maybe we can rephrase this to something like:
> When resolving the module-path, setting this value to true will include the dependencies that generate path exceptions in the classpath. By default the value is false, and these dependencies won't be included.

**Comment by atdixon** [comment]
I updated to your phrasing--thank you.

**Comment by jperedadnr** [comment]
Perfect, thanks. One final step before we can merge: Can you please fill the [Gluon Individual Contributor License Agreement (CLA)](https://docs.google.com/forms/d/16aoFTmzs8lZTfiyrEm8YgMqMYaGQl0J8wA0VJE2LCCY)?
